src/foam/dao/jdbc/ConnectionPool.js

```diff
@@ -83,9 +83,7 @@ foam.CLASS({
   pool.setUrl(getConnectionString());
   pool.setInitialSize(getPoolSize());
   setPool(pool);
-} catch (Exception e) {
-  e.printStackTrace();
-}`
+} catch (Exception e) {}`
```

Comment:
Can we not log these?  I don't like exceptions that are just eaten.

Reply:
Should I log it with logger.error() method or System.out.println?